js/cart.js

```diff
@@ -0,0 +1,143 @@
+const products = JSON.parse(localStorage.getItem("products"));
+let cart = JSON.parse(localStorage.getItem('cart'));
+const storePage = document.getElementById('store-page');
+const cartNumber = document.getElementById("cart-number");
+
+function renderCartNumber() {
+  const quantityProduct = cart.reduce((acc, item) => acc + item.quantity, 0);
+  if (quantityProduct !== 0) {
+    cartNumber.classList.add("cart-active");
+    cartNumber.innerHTML = quantityProduct;
+  } else {
+    cartNumber.classList.remove("cart-active");
+    cartNumber.innerHTML = "";
+  }
+}
+
+function renderCart() {
+  if(cart.length > 0) {
+    storePage.innerHTML = `
+    <section class="section section-store">
+      <div class="container">
+        <div class="row store-inner">
+          <div class="col-8 col-sm-12">
+            <h3 class="txt-bold txt-center cart-title">Shopping Cart</h3>
+            <ul class="cart-list" id="cart-list">
+              <div class="cart-row">
+                <span class="txt-bold cart-body">PRODUCT</span>
+                <span class="txt-bold cart-buttons">QUANTITY</span>
+                <span class="txt-center txt-bold cart-total">TOTAL</span>
+                <span class="cart-close"></span>
+              </div>
+            ${cart.map((item) => {
+              return `                
+              <li class="cart">
+                <div class="cart-body">
+                  <div class="cart-image">
+                    <img
+                      src="${item.image}"
+                      alt="${item.name}"
+                    />
+                  </div>
+                  <div class="cart-content">
+                    <h4 class="typo-2 txt-light cart-name">
+                      ${item.name}
+                    </h4>
+                    <span class="txt-regular cart-price">$${(item.price - (item.price * item.discount / 100)).toFixed(2)}</span>
+                  </div>
+                </div>
+                <div class="cart-buttons">
+                  <button class="cart-btn" onclick={minusCart(${item.id})}>
+                    <i class="bx bx-minus"></i>
+                  </button>
+                  <div class="cart-quantity">${item.quantity}</div>
+                  <button class="cart-btn" onclick={addCart(${item.id})}>
+                    <i class="bx bx-plus"></i>
+                  </button>
+                </div>
+                <p class="txt-center cart-total">
+                $${((item.price - (item.price * item.discount / 100)) * item.quantity).toFixed(2)}
```

Comment:
nếu em để ý toàn page đều chung một `fixed`, ở đây em khai báo một biến `fixed` và một hàm riêng chịu trách nhiệm `format` giá trị